source/content/guides/drupal-9/drupal-9-hosted/01-introduction.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Migrate a Drupal 8 Site to Drupal 9
+subtitle: Introduction
+description: 
+cms: "Drupal 9"
+categories: [develop]
+tags: [code, launch, migrate, site, updates]
+contributors: [wordsmither]
+reviewed: "2021-03-31"
+layout: guide
+showtoc: true
+permalink: docs/guides/drupal-9-hosted
+anchorid: drupal-9-hosted
+editpath: drupal-9/drupal-9-hosted/01-introduction.md
+---
+
+This guide will show you how to migrate a site that meets the following criteria to Drupal 9:
+
+|Criteria|Value
+|---|---
+|**Pantheon Hosted**| **Yes**
+|Site created using:| n/a
+|**Multi-Dev Environment** | **No**
+|Build Tools Workflow Needed | n/a
+|Drupal version set to 8| n/a
+|Site created before November 2021| n/a
+
+<Partial file="drupal-9/commit-history.md" />
+
+This doc uses the following aliases:
+
+- **Alias:** `D8_SITE`
+  - **Site Name:** `best-drupal8-site-ever`
+- **Alias:** `D9_SITE`
+  - **Site Name:** `best-drupal9-site-ever`
+
+## Requirements
+
+<Partial file="drupal-9/upgrade-site-requirements-new.md" />
```

Comment:
When used to migrate from empty, this should reflect empty upstream instead of drops-8